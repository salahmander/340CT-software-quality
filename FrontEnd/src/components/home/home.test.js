import React from 'react';
import { shallow } from 'enzyme';
import ReactDOM from 'react-dom';

import Home from './home'

describe('item list ', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Home />, div);
        ReactDOM.unmountComponentAtNode(div);
      });

    it('should render correctly in "debug" mode and match the snapshot', () => {
        const component = shallow(<Home debug />);
        expect(component).toMatchSnapshot();
      });
      
    });