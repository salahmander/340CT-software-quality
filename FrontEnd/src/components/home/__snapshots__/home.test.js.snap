// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`item list  should render correctly in "debug" mode and match the snapshot 1`] = `
<div
  className="cf"
>
  <img
    alt="laptop"
    className="top"
    src="firstImage.png"
  />
  <img
    alt="laptop"
    className="bottom"
    src="secondImage.png"
  />
</div>
`;
